website/docs/tutorials/03-project-initialization.md

```diff
@@ -29,10 +29,10 @@ The result of running `protostar init` is a configuration file `protostar.toml`,
 ### Project configuration
 ```toml title="'protostar.toml' is required"
 ["protostar.config"]
-protostar_version = "0.1.0"
+protostar_version = "0.5.0"
```

Comment:
Is it possible to populate this in docs build time, reading from `pyproject.toml`?

Reply:
Hmm, it should be possible. We could use keep protostar version in the `docusaurus.config.js` and update that file on build time.

Reply:
If you can put it in `docusaurus.config.js` then you cal also read it from `pyproject.toml`. My general motivation is to keep version number in one place.

Reply:
https://github.com/software-mansion/protostar/issues/951